akka-cluster-sharding-typed/src/test/scala/akka/cluster/sharding/typed/JoinConfigCompatCheckerClusterShardingSpec.scala

```diff
@@ -82,7 +79,12 @@ class JoinConfigIncompatibilitySpec extends JoinConfigCompatCheckerClusterShardi
   }
 }
 
-class JoinConfigCompatibilitySpec extends JoinConfigCompatCheckerClusterShardingSpec {
+class JoinConfigCompatibilitySpec
+    extends JoinConfigCompatCheckerClusterShardingSpec(
+      // increased to shut down two actor systems, which intermittently takes longer in CI,
+      // vs one system in JoinConfigIncompatibilitySpec
+      ConfigFactory.parseString("akka.actor.testkit.typed.system-shutdown-default = 15s")) {
```

Comment:
I can find no reason, other than sporadically needing this in CI for two vs one shutdown, but no proof. Can not reproduce the failure.